composer.json

```diff
@@ -3,8 +3,10 @@
     "type": "neos-package",
     "name": "punktde/mautic",
     "license": "MIT",
+    "version": "1.1",
     "require": {
-        "neos/neos": "^4.0",
+        "neos/neos": "^5.0",
+        "neos/form-builder": "^2.0",
```

Comment:
Were there an actual dependency problem with form-builder? Since Mautic is usable without forms (even it doesn't make any sense without forms => no forms = no quality leads with contact data)

imo this is ok